### DISCUSSION TITLE
Authorization Policy - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3734] [Announcements]
# Authorization Policy - DaoCloud Enterprise

DaoCloud Enterprise 5.0 is the next-generation PaaS platform, created with many leading cloud-native technologies, to easily promote and boost your digital business.

https://docs.daocloud.io/en/mspider/user-guide/security/authorize.html

**Comment by hanxiaop** [comment]
Missing images like https://docs.daocloud.io/mspider/user-guide/security/authorize.html

**Reply by aleung939** [reply]
Missing image in English https://docs.daocloud.io/daocloud-docs-images/docs/en/docs/images/09cloud-edge-update.png
only Chinese Version is available: https://docs.daocloud.io/daocloud-docs-images/docs/zh/images/09cloud-edge-update.png

**Reply by windsonsea** [reply]
fixed
